pom.xml

```diff
@@ -33,6 +33,8 @@
         <!-- Docker -->
         <docker.jib-maven-plugin.version>1.3.0</docker.jib-maven-plugin.version>
         <docker.image.prefix>springcommunity</docker.image.prefix>
+        <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+        <spring-boot-maven-plugin.version>2.6.2</spring-boot-maven-plugin.version>
```

Comment:
This `pom.xml` inherits from the `spring-boot-starter-parent` thus I don't think we have to declare the `spring-boot-maven-plugin` version.